components/data-display/component-preview.tsx

```diff
@@ -75,7 +146,45 @@ export const ComponentPreview = memo(
         return props
       }, [_containerProps])
 
-      return (
+      return iframe ? (
+        <ui.iframe
+          title="component-preview-iframe"
+          ref={iframeRef}
+          w="full"
+          minH="md"
+          h="full"
+          display={rest.display}
+        >
+          {head && body
+            ? createPortal(
+                <>
+                  <CacheProvider value={createCache(head)}>
+                    <CustomUIProvider {...value}>
+                      <Center
+                        ref={ref}
+                        flexDirection="column"
+                        boxSize="full"
+                        minH="48"
+                        {...rest}
+                      >
+                        {!loading ? (
+                          <Box boxSize="full" flex="1" {...containerProps}>
+                            <Component />
+                          </Box>
+                        ) : (
+                          <Center boxSize="full" flex="1">
+                            <Loading size="6xl" {...loadingProps} />
+                          </Center>
+                        )}
+                      </Center>
+                    </CustomUIProvider>
+                  </CacheProvider>
+                </>,
```

Comment:
Does this need to be wrapped inside `<></>`?
```suggestion
                <CacheProvider value={createCache(head)}>
                  <CustomUIProvider {...value}>
                    <Center
                      ref={ref}
                      flexDirection="column"
                      boxSize="full"
                      minH="48"
                      {...rest}
                    >
                      {!loading ? (
                        <Box boxSize="full" flex="1" {...containerProps}>
                          <Component />
                        </Box>
                      ) : (
                        <Center boxSize="full" flex="1">
                          <Loading size="6xl" {...loadingProps} />
                        </Center>
                      )}
                    </Center>
                  </CustomUIProvider>
                </CacheProvider>,
```

Reply:
I don't think that is necessary either.